lib/custom_exceptions.py

```diff
@@ -0,0 +1,14 @@
+class TenableScanRunningException(Exception):
+    pass
```

Comment:
What is the purpose of these?

Reply:
They are the custom exceptions I use to control the state machine, in order to perform conditional branching / retries. For instance, if a function raises `TenableScanRunningException` the state machine waits for a period of time before checking again.

See [this](https://github.com/mozilla/vautomator-serverless/blob/04355d127cb6c71b6518ae84a6c3fced0c8ed719/serverless.yml#L356-L369) as to how these are used.